src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsVuapFandomMobile.java

```diff
@@ -0,0 +1,126 @@
+
+        videoFanTakeover.clickOnAdImage();
+
+        String tabUrl = fandomPage.switchToNewBrowserTab();
+        videoFanTakeover.verifyFandomTabOpened(tabUrl);
+    }
+
+    @Test(
+            dataProviderClass = FandomAdsDataProvider.class,
+            dataProvider = "fandomVuapPageMobile",
+            groups = "AdsVuapVideoClosesWhenTapCloseButtonFandomMobile"
+    )
+    public void adsVuapVideoClosesWhenTapCloseButtonFandomMobile(String pageType, String pageName, String slotName, String iframeId) {
+        AdsFandomObject fandomPage = loadPage(pageName, pageType);
+        VideoFanTakeover videoFanTakeover = prepareSlot(slotName, iframeId, fandomPage);
+
+        videoFanTakeover.play();
+
+        videoFanTakeover.waitForVideoStart(slotName);
+
+        videoFanTakeover.clickOnVideoCloseButton();
+
+        videoFanTakeover.waitForVideoPlayerHidden(slotName);
+    }
+
+    @NetworkTrafficDump
+    @Test(
+            dataProviderClass = FandomAdsDataProvider.class,
+            dataProvider = "fandomVuapPageMobile",
+            groups = "AdsVuapTimeProgressingFandomMobile"
+    )
+    public void adsVuapTimeProgressingFandomMobile(String pageType, String pageName, String slotName, String iframeId) throws InterruptedException {
+        networkTrafficInterceptor.startIntercepting();
+        AdsFandomObject fandomPage = loadPage(pageName, pageType);
+        VideoFanTakeover videoFanTakeover = prepareSlot(slotName, iframeId, fandomPage);
+
+        videoFanTakeover.play();
+
+        videoFanTakeover.waitForVideoStart(slotName);
+        fandomPage.wait.forSuccessfulResponse(networkTrafficInterceptor, URL_FIRSTQUARTILE);
+        double quartileTime = videoFanTakeover.getCurrentVideoTimeOnMobile(slotName).doubleValue();
+
+        fandomPage.wait.forSuccessfulResponse(networkTrafficInterceptor, URL_MIDPOINT);
+        double midTime = videoFanTakeover.getCurrentVideoTimeOnMobile(slotName).doubleValue();
+        Assertion.assertTrue(videoFanTakeover.isTimeProgressing(quartileTime, midTime));
+    }
+
+    @NetworkTrafficDump
+    @Test(
+            dataProviderClass = FandomAdsDataProvider.class,
+            dataProvider = "fandomVuapPageMobile",
+            groups = "AdsVuapVideoPauseFandomMobile"
+    )
+    public void adsVuapVideoPauseFandomMobile(String pageType, String pageName, String slotName, String iframeId) throws InterruptedException {
+        networkTrafficInterceptor.startIntercepting();
+        AdsFandomObject fandomPage = loadPage(pageName, pageType);
+        VideoFanTakeover videoFanTakeover = prepareSlot(slotName, iframeId, fandomPage);
+
+        videoFanTakeover.play();
+
+        videoFanTakeover.waitForVideoStart(slotName);
+        fandomPage.wait.forSuccessfulResponse(networkTrafficInterceptor, URL_FIRSTQUARTILE);
+
+        videoFanTakeover.pause();
+
+        double time = videoFanTakeover.getCurrentVideoTimeOnMobile(slotName);
+
+        TimeUnit.SECONDS.sleep(DELAY);
```

Comment:
This sleep has been created to check if after pause video, video time did not changed after two seconds we can not use any other wait which will replace sleep :( because nothing change in layout and we want to make sure that video time has not change after couple seconds